src/qml/components/AboutOptions.qml

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import "../controls"
+
+ColumnLayout {
+    spacing: 20
+    Information {
+        Layout.fillWidth: true
+        header:  qsTr("Website")
+        description: qsTr("bitcoincore.org >")
```

Comment:
The descriptions here should not be translatable.

Reply:
Addressed in #140
Thanks!